outputs.tf

```diff
@@ -133,6 +133,11 @@ output "database_subnet_group" {
   value       = concat(aws_db_subnet_group.database.*.id, [""])[0]
 }
 
+output "database_subnet_group_name" {
+  description = "Name of database subnet group"
+  value       = aws_db_subnet_group.database.*.name[0]
```

Comment:
do we need the `[0]` at the end?

Reply:
We don't want a list here, but a string. And ss the database subnet group name [is the same](https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/main.tf#L411) whatever the number of subnets is defined, it is safe to take the first occurrence on the list.